src/main/resources/edu/cornell/kfs/pdp/businessobject/datadictionary/ACHPayee.xml

```diff
@@ -58,8 +58,43 @@
             </list>
           </property>
         </bean>
+        
+        <!-- Add "principalName" to the ACHPayee lookup results. -->

```

Comment:
What I meant before was that the XML comments were making distinctions between input field overrides and result field overrides; sorry if that wasn't clear. Some reordering or additional comments are needed to get things to line up; alternatively, you could just remove the input field and result field comments if you think the beans are self-explanatory enough.

Reply:
updating